chart/templates/gateway-deployment.yaml

```diff
@@ -55,3 +67,7 @@ spec:
           value: http://{{ .Release.Name }}-vsp/
         - name: PROXY_NODE_ERROR_PAGE_URI
           value: {{ .Values.gateway.errorPageURL }}
+        - name: PROXY_NODE_METADATA_RESOURCE_FILE_PATH
```

Comment:
does `PROXY_NODE_METADATA_RESOURCE_FILE_PATH ` and `PROXY_NODE_METADATA_PUBLISH_PATH ` really need to be configurable via the environment?.... looks like they could just be hardcoded or could they have decent defaults so that the amount of config could be reduced.

Reply:
Yeah maybe they could - I'm still trying to figure if this will work and will refine this further